src/databricks/labs/ucx/hive_metastore/tables.py

```diff
@@ -54,6 +54,8 @@ class Table:
         "dbfs:/databricks-datasets",
     ]
 
+    UPGRADED_FROM_WS_PARAM: typing.ClassVar[str] = "upgraded_from_ws"
```

Comment:
```suggestion
    UPGRADED_FROM_WS_PARAM: typing.ClassVar[str] = "upgraded_from_workspace_id"
```
ws is not clear for users

Reply:
Fixed